db/migrate/20170127094357_add_caching_enabled_to_compute_resource.rb

```diff
@@ -0,0 +1,5 @@
+class AddCachingEnabledToComputeResource < ActiveRecord::Migration
+  def change
+    add_column :compute_resources, :caching_enabled, :boolean
```

Comment:
do you see a need to default to false? I would rather opt in by default, no one wants the slower option unless they have a problem?